client/components/ZoopsShow.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+
+const ZoopsShow = () => {
```

Comment:
nit: naming here also feels a little non-standard to me - I would expect something like ZoopDetail or SingleZoopView or SingleZoopPage

Reply:
I think the plural in the name is especially throwing me

Reply:
FYI I had made suggestions for components and their naming in the Jira (these are all just suggestions though, it's ultimately up to you both to decide on things like naming conventions)

Reply:
I'll take a look at those. Agreed that "ZoopsView" is a little confusing cause of the plural.

Reply:
Yeah I think the plural there was a mistake. Definitely not married to the naming in any case. Index and Show were just what App Academy taught us